python/GafferUSDTest/USDShaderTest.py

```diff
@@ -99,8 +102,8 @@ def testLoadUsdPreviewSurface( self ) :
 				( "occlusion", Gaffer.FloatPlug, 1 ),
 			],
 			[
-				( "surface", Gaffer.Plug, None ),
-				( "displacement", Gaffer.Plug, None ),
+				( "surface", Gaffer.Plug if legacyUsdVersion else Gaffer.StringPlug, None if legacyUsdVersion else "" ),
+				( "displacement", Gaffer.Plug if legacyUsdVersion else Gaffer.StringPlug, None if legacyUsdVersion else "" ),
```

Comment:
I'm going to spend a bit of time today to see if there might be some way of keeping the original behaviour. I'll hold off merging pending the results of that.